react/features/app/components/App.native.js

```diff
@@ -1,7 +1,6 @@
 /* global __DEV__ */
 
 import React from 'react';
-import { Linking } from 'react-native';
```

Comment:
If react-native's `Linking` is removed, shouldn't https://github.com/jitsi/jitsi-meet/blob/master/ios/Podfile#L13 i.e. `RCTLinkingIOS` also be removed?

Reply:
No because we still use it to get the initial arguments and to open URLs with the native browser. A more accurate description would be that we removed the need for the Linking module for handling deep / universal links. My bad!